addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -19,99 +17,170 @@
+                                        t-attf-class="p-0 text-body #{country['active'] and ' active' or ''}">
+                                            <div class="form-check">
+                                                <input class="form-check-input" type="radio" t-attf-name="country['country_id'][1] offcanvas" t-att-checked="country['active'] and true or false"/>
+                                                <label class="form-check-label" t-attf-for="country['country_id'][1] offcanvas"><t t-esc="country['country_id'][1]"/></label>
+                                            </div>
+                                        </a>
+                                    </li>
+                                </t>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
             </div>
-            <div class='navbar'>
+        </aside>
+        <div class="row mb-4" id="ref_content">
+            <div class="my-5 py-5 text-center" t-if="not partners">
+                <h5>No results found for "<span t-out="searches.get('search', '')"/>"</h5>
+                <a href="/partners">See all resellers</a>
+            </div>
+            <t t-foreach="partners" t-as="partner">
+                <t t-if="last_grade != partner.grade_id.id">
+                    <h5 class="mt-4 mb-2 col-12">
+                        <span t-field="partner.grade_id"/> Partners
+                        <t t-call="website.publish_management">
+                            <t t-set="object" t-value="partner.grade_id"/>
+                            <t t-set="publish_edit" t-value="True"/>
+                        </t>
+                    </h5>
+                    <t t-set="last_grade" t-value="partner.grade_id.id"/>
+                </t>
+                <div class="col-md-4 col-xl-3 col-12 mb-4">
+                    <div class="card text-decoration-none h-100">
+                        <a t-field="partner.image_1920"
+                            t-attf-href="/partners/#{slug(partner)}?#{current_grade and 'grade_id=%s&amp;' % current_grade.id}#{current_country and 'country_id=%s' % current_country.id}"
+                            class="card-img-top border-bottom"
+                            t-options='{"widget": "image", "qweb_img_responsive": False, "class": "img img-fluid h-100 w-100 mw-100", "style": "max-height: 208px; min-height: 208px; object-fit: cover"}'/>
+                        <div class="card-body">
+                            <h5 class="card-title" t-attf-href="/partners/#{slug(partner)}?#{current_grade and 'grade_id=%s&amp;' % current_grade.id}#{current_country and 'country_id=%s' % current_country.id}">
+                                <span t-field="partner.display_name"/>
+                            </h5>
+                            <div id="address">
```

Comment:
The `address` id name seems too generic, could you prefix it?
If the `<div>` is filled in JS (I think it's the case) don't forget to adapt the JS accordingly